draft-ietf-scitt-architecture.md

```diff
@@ -214,26 +221,26 @@ Issuer:
 : an entity that creates Signed Statements about software Artifacts in the supply chain.
 An Issuer may be the owner or author of software Artifacts, or an independent third party such as a reviewer or an endorser.
 
+Ledger (was Registry):
+
+: the verifiable append-only data structure that stores Signed Statements in a Transparency Service.
+SCITT supports multiple Ledger and Receipt formats to accommodate different Transparency Service implementations, such as historical Merkle Trees and sparse Merkle Trees.
```

Comment:
```suggestion
```

Reply:
I think this is ledger specific implementation detail, not "cross ledger" or "SCITT".

Reply:
@OR13, while I agree, I'm not sure what you're suggesting we'd change here. I was reading this as how SCITT could operate, acknowledging different implemetnations are still consistent with a SCITT spec.